### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Widcket**
### Changes

- Removed outdated info.
- Mentioned new features.
- Updated links.
- Synced the structure with Auth0.swift's [README](https://github.com/auth0/Auth0.swift/blob/master/README.md).

### Checklist

[X] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
[X] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
[X] All existing and new tests complete without errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
...
+Just clone the repo, run `carthage bootstrap` and you're ready to contribute!
 
-## Issue Reporting
+## What is Auth0?
 
-If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+Auth0 helps you to:
 
-## License
+* Add authentication with [multiple authentication sources](https://auth0.com/docs/identityproviders), either social like **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce, amont others**, or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS or any SAML Identity Provider**.
```
Comment (stevehobbsdev) [inline README.md]: Seems redundant to mention 'authentication' twice here. Also a couple of edits and typos:

```suggestion
* Add authentication with [multiple sources](https://auth0.com/docs/identityproviders), either social identity providers such as **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce** (amongst others), or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS, or any SAML Identity Provider**.
```

File: README.md [hunk 2]
```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
...
+* Add authentication through more traditional **[username/password databases](https://auth0.com/docs/connections/database/custom-db)**.
+* Add support for **[linking different user accounts](https://auth0.com/docs/link-accounts)** with the same user.
+* Support for generating signed [JSON Web Tokens](https://auth0.com/docs/tokens/concepts/jwts) to call your APIs and **flow the user identity** securely.
+* Analytics of how, when and where users are logging in.
```
Comment (stevehobbsdev) [inline README.md]:
```suggestion
* Analytics of how, when, and where users are logging in.
```

File: README.md [hunk 3]
```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
...
+* Pull data from other sources and add it to the user profile, through [JavaScript rules](https://auth0.com/docs/rules).
```
Comment (stevehobbsdev) [inline README.md]:
```suggestion
* Pull data from other sources and add it to the user profile through [JavaScript rules](https://auth0.com/docs/rules).
```

File: README.md [hunk 4]
```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
...
 
-SimpleKeychain is available under the MIT license. See the [LICENSE file](https://github.com/auth0/SimpleKeychain/blob/master/LICENSE) for more info.
+## Create a Free Auth0 Account
 
-## Author
+1. Go to [Auth0](https://auth0.com) and click Sign Up.
```
Comment (stevehobbsdev) [inline README.md]:
```suggestion
1. Go to [Auth0](https://auth0.com) and click **Sign Up**.
```

File: README.md [hunk 5]
```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
...
+2. Use Google, GitHub or Microsoft Account to login.
```
Comment (stevehobbsdev) [inline README.md]:
```suggestion
2. Use Google, GitHub, or Microsoft Account to login.
```

File: README.md [hunk 6]
```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
...
 
-[Auth0](https://auth0.com)
+## Issue Reporting
 
-## What is Auth0?
+If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/responsible-disclosure-policy) details the procedure for disclosing security issues.
```
Comment (stevehobbsdev) [inline README.md]: I'd make this slightly more concise and link them directly to issues for ease:

```suggestion
If you have found a bug or to request a feature, please [raise an issue](https://github.com/auth0/simplekeychain/issues). Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/responsible-disclosure-policy) details the procedure for disclosing security issues.
```

**Comment by Widcket** [comment]
Thanks @stevehobbsdev!

**Pull request merged**